ocaml/lambda/tmc.ml

```diff
@@ -62,7 +62,12 @@ and offset = Offset of lambda
 let offset_code (Offset t) = t
 
 let add_dst_params ({var; offset} : Ident.t destination) params =
-  (var, Lambda.layout_block) :: (offset, Lambda.layout_int) :: params
+  (* CR ncourant: are these modes necessarily heap? *)
```

Comment:
Yes, the result of a tmc function is always heap, so these modes are correct. (If you try to make a local-returning tmc function, you'll get an error in `Tmc.find_candidate`).